wgsl/index.bs

```diff
@@ -9384,6 +9391,12 @@ Each is described in detail in subsequent sections.
       <td rowspan=2>u32
       <td rowspan=2>[=extension/subgroups=]
   <tr><td>fragment
+
+  <tr><td>[=built-in values/primitive_id=]
```

Comment:
I feel like this should be between frag_depth and sample_index.

Reply:
Done